[prometheus-redis-exporter] support multiple targets (#2199)
prometheus-community/helm-charts: charts/prometheus-redis-exporter/README.md

_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._

### To 5.0.0

> From 5.0.0 redis exporter is using the [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). Therfore youl have to delete the deployment before your upgrade.

Reviewer: Sorry, some typos...

```suggestion
From 5.0.0 redis exporter is using the [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). Therefore you have to delete the deployment before you upgrade.
```

_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._

### To 5.0.0

> From 5.0.0 redis exporter is using this above as selector labels in service and deployments

Reviewer: ```suggestion
From 5.0.0 redis exporter is using the [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). Therfore youl have to delete the deployment before your upgrade.
```

> ```

Reviewer: Line 56 - 62 can be deleted.

Author: It is deleted now.

app.kubernetes.io/name: {{ include "prometheus-redis-exporter.name" . }}
app.kubernetes.io/instance: {{ .Release.Name }}
```

selector labels is immutable in kubernetes, so uninstall is needed if you're upgrading from 4.8.0 and below

```console

> helm uninstall {{ .Release.Name }}

Reviewer: You could keep your helm history if you delete the deployment before upgrading:

```suggestion
kubectl delete deployment -l app=prometheus-redis-exporter
helm upgrade -i prometheus-redis-exporter prometheus-community/prometheus-redis-exporter
```

Author: Thank you! I didn't know that could be done
